templates/python/HelloWorld/gdk-config.json

```diff
@@ -13,4 +16,4 @@
     }
   },
   "gdk_version": "1.0.0"
```

Comment:
update minimum gdk version for the zip_name feature

Reply:
Updated with version 1.3.0